picard/util/__init__.py

```diff
@@ -426,8 +426,8 @@ def tracknum_and_title_from_filename(base_filename):
         tnlen = len(tracknumber)
         if stripped_filename[:tnlen] == tracknumber:
             title = stripped_filename[tnlen:].lstrip()
-
-    return (tracknumber, title)
+    GuessedFromFilename = namedtuple('GuessedFromFilename', ('tracknumber', 'title'))
```

Comment:
Should this named toupee rather be defined outside of the function? It seems odd the redefine it on every call

Reply:
Yes, you're right, done in dc0aeb8